src/redux/user/actions.js

```diff
@@ -26,6 +26,33 @@ export const loadWritePermissions = (getUserPermissions) => (dispatch) => {
     );
 };
 
+export const loadReadIntegrationsEndpointsPermissions = (getUserPermissions) => (dispatch) => {
+  dispatch({ type: ACTION_TYPES.SET_READ_INTEGRATIONS_ENDPOINTS_PERMISSIONS_PENDING });
+
+  return getUserPermissions('integrations', true) // bypassCache = true
+    .then((permissions) => {
+      const allPermission = permissions.reduce((acc, curr) => [...acc, curr?.permission], []);
```

Comment:
No need to use reduce and copy object.
```suggestion
      const allPermission = permissions.map((curr) => curr?.permission);
```